protocol/testing/containertest/containertest.sh

```
@@ -8,6 +8,7 @@ source "./genesis.sh"
 source "./version.sh"
 
 CHAIN_ID="localdydxprotocol"
+PREUPGRADE_VERSION="v8.0.0-rc1"
```

Shellcheck (0.10.0)

[warning] PREUPGRADE_VERSION appears unused. Verify use (or export if used externally).

(SC2034)